sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -688,7 +683,6 @@ def get_paths(
         :returns: An iterable (auto-paging) response of PathProperties.
         :rtype: ~azure.core.paging.ItemPaged[~azure.storage.filedatalake.PathProperties]
         """
-        timeout = kwargs.pop('timeout', None)
```

Comment:
This `timeout` `kwargs` popping line should be removed - `timeout` is already a named kwarg!